.buildkite/scripts/ech.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+function ech_up() {
+  echo "~~~ Starting ECH Stack"
+  local STACK_VERSION=$1
+
+  if [ -z "$STACK_VERSION" ]; then
+    echo "Error: Specify stack version: ech_up [stack_version]" >&2
+    return 1
+  fi
+
+  oblt-cli cluster create custom \
+      --template ech-beats-it \
```

Comment:
This template probably does not exist. `oblt-cli` uses templates to delegate what services need to run and reuse it.

I can create a new template if you tell me what part of the Elastic Stack you need to run. Or you can use`ess-ea-it` for now - it configures an oblt cluster that runs on ESS dev region on production with ES, Kibana, and Integrations.